src/test/java/org/mockitousage/verification/VerificationWithTimeoutTest.java

```diff
@@ -137,21 +156,32 @@ public void shouldAllowMixingOnlyWithTimeoutAndFail() throws Exception {
      * is used.
      */
     @Test
-    public void canIgnoreInvocationsWithJunit() {
+    public void canIgnoreInvocationsWithJunit() throws InterruptedException {
+        // given
+        DelayedExecution delayedExecution = new DelayedExecution(executor, mock, 0);
+        delayedExecution.recordAsyncCall('0');
+        delayedExecution.recordAsyncCall('1');
+        delayedExecution.recordAsyncCall('2');
+
         // when
-        callAsyncWithDelay(mock, '0', 0, MILLISECONDS);
-        callAsyncWithDelay(mock, '1', 0, MILLISECONDS);
-        callAsyncWithDelay(mock, '2', 20, MILLISECONDS);
+        delayedExecution.allAsyncCallsStarted();
 
         // then
+        verify(mock, timeout(50)).oneArg('0');
```

Comment:
@ChristianSchwarz Good point. I overlooked this one thinking the countdown latch semantic forbid that, but there is two way to handle this 
* it's possible add a new instance of `DelayedExecution` and insert `allAsyncCallsStarted` before the last verify
* tweak the `DelayedExecution` API to be more fine grained

About the missing verify I noted it as well in the original test as well, but it may be de to a human error so I'm ok with the addition.